raft/src/test/java/org/apache/kafka/raft/RaftEventSimulationTest.java

```diff
@@ -329,7 +330,9 @@ private EventScheduler schedulerWithDefaultInvariants(Cluster cluster) {
         EventScheduler scheduler = new EventScheduler(cluster.random, cluster.time);
         scheduler.addInvariant(new MonotonicHighWatermark(cluster));
         scheduler.addInvariant(new MonotonicEpoch(cluster));
-        scheduler.addInvariant(new ConsistentCommittedData(cluster));
+        scheduler.addInvariant(new MajorityReachedHighWatermark(cluster));
+        scheduler.addInvariant(new SingleLeader(cluster));
```

Comment:
nit: add one line space between L334 and L335.

Reply:
I tried this out, but I felt it made the function look a little odd.
```java
    private EventScheduler schedulerWithDefaultInvariants(Cluster cluster) {
        EventScheduler scheduler = new EventScheduler(cluster.random, cluster.time);
        scheduler.addInvariant(new MonotonicHighWatermark(cluster));
        scheduler.addInvariant(new MonotonicEpoch(cluster));
        scheduler.addInvariant(new MajorityReachedHighWatermark(cluster));
        scheduler.addInvariant(new SingleLeader(cluster));

        scheduler.addValidation(new ConsistentCommittedData(cluster));
        return scheduler;
    }
```
I guess we could go crazy with spacing and do something like this:
```java
    private EventScheduler schedulerWithDefaultInvariants(Cluster cluster) {
        EventScheduler scheduler = new EventScheduler(cluster.random, cluster.time);
       
        scheduler.addInvariant(new MonotonicHighWatermark(cluster));
        scheduler.addInvariant(new MonotonicEpoch(cluster));
        scheduler.addInvariant(new MajorityReachedHighWatermark(cluster));
        scheduler.addInvariant(new SingleLeader(cluster));

        scheduler.addValidation(new ConsistentCommittedData(cluster));

        return scheduler;
    }
```
But that seems excessive. If there were more validations, it would look nicer. Anyway, this is all subjective.